runtime/cfdumper/main.c

```diff
@@ -2304,7 +2304,7 @@ static U_32 buildFlags(void)
 	/* Set vm flags to the latest version to pass version validation
 	 * check in j9bcutil_readClassFileBytes.
 	 */
-	flags |= BCT_JavaMaxMajorVersionShifted;
+	flags |= BCT_JavaMaxMajorVersionShifted | BCT_AnyPreviewVersion | BCT_EnablePreview;
```

Comment:
Does this need both `BCT_AnyPreviewVersion | BCT_EnablePreview`?  I would have expected `BCT_AnyPreviewVersion` to be sufficient

Reply:
True.  Fixed.